go.mod

```diff
@@ -9,16 +9,16 @@ require (
 	github.com/go-logr/logr v0.4.0
 	github.com/go-sql-driver/mysql v1.5.0 // indirect
 	github.com/gruntwork-io/terratest v0.37.7
-	github.com/k8ssandra/cass-operator v1.7.1
+	github.com/k8ssandra/cass-operator v1.8.0-rc2
 	github.com/pkg/errors v0.9.1
 	github.com/rs/zerolog v1.20.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
 	gopkg.in/resty.v1 v1.12.0
-	k8s.io/api v0.21.2
-	k8s.io/apimachinery v0.21.2
+	k8s.io/api v0.21.4
```

Comment:
These values are overridden in the replace directive. Should be the same k8s version for everything.

Reply:
I have no clue why/how these were changed 🤔

Reply:
fixed